modules/sdk/project-templates/project-templates-form-field/build.gradle

```diff
@@ -2,12 +2,22 @@ dependencies {
 	compileOnly group: "com.beust", name: "jcommander", version: "1.72"
 	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.17"
 	compileOnly group: "org.apache.maven.archetype", name: "archetype-common", version: "2.4"
+	compileOnly project(":core:petra:petra-string")
```

Comment:
hey @gabrielibson Lets not add a dependency here to petra-string. We try to keep the project-templates customizers pretty self contained. Just copy whatever strings from StringPool that you need.